<!DOCTYPE html>
<html>

  <head>
    <meta charset='utf-8' />
    <meta http-equiv="X-UA-Compatible" content="chrome=1" />
    <meta name="description" content="PeptideEclipse : Map your peptide identifications to UniProt transmembrane regions!" />

    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">

    <title>PeptideEclipse</title>
  </head>

  <body>

    <!-- HEADER -->
    <div id="header_wrap" class="outer">
        <header class="inner">
          <a id="forkme_banner" href="https://github.com/ulo/PeptideEclipse">View on GitHub</a>

          <h1 id="project_title">PeptideEclipse</h1>
          <h2 id="project_tagline">Map your peptide identifications to UniProt transmembrane regions!</h2>

            <section id="downloads">
              <a class="zip_download_link" href="https://github.com/ulo/PeptideEclipse/zipball/master">Download this project as a .zip file</a>
              <a class="tar_download_link" href="https://github.com/ulo/PeptideEclipse/tarball/master">Download this project as a tar.gz file</a>
            </section>
        </header>
    </div>

    <!-- MAIN CONTENT -->
    <div id="main_content_wrap" class="outer">
      <section id="main_content" class="inner">
        <h3>
<a name="welcome-to-peptideeclipse" class="anchor" href="#welcome-to-peptideeclipse"><span class="octicon octicon-link"></span></a>Welcome to PeptideEclipse</h3>

<p>PeptideEclipse allows you to map your proteomics peptide identifications to transmembrane regions as annotated in UniProt, 
and thus showing which peptides were completely or partially hidden within the lipid bilayer.</p>

<p>Homepage: <a href="http://ulo.github.io/PeptideEclipse/">http://ulo.github.io/PeptideEclipse/</a></p>

<h3>
<a name="input" class="anchor" href="#input"><span class="octicon octicon-link"></span></a>Input</h3>

<p>First, download the corresponding UniProt Knowledgebase packages from their FTP server:</p>

<p><a>ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/</a></p>

<p>Note: you do not have to unzip the files!</p>

<p>Then just provide the ProteinProphet output, either the complete <code>prot.xml</code> file or a filtered and exported <code>prot.xls</code> file.</p>

<h3>
<a name="command" class="anchor" href="#command"><span class="octicon octicon-link"></span></a>Command</h3>

<p>To run PeptideEclipse type the following into your command line:</p>

<pre><code>java -jar PeptideEclipse.jar -in inFile -up uniprot1.dat.gz [uniprot2.dat.gz ...] &gt; outFile
</code></pre>

<ul>
<li>inFile: a ProteinProphet file (e.g: interact.prot.xml | interact.prot.xls)</li>
<li>outFile: target file (e.g: interact.prot.tmAnalysis.tsv)</li>
</ul><h3>
<a name="output" class="anchor" href="#output"><span class="octicon octicon-link"></span></a>Output</h3>

<p>For a <code>prot.xml</code> file input, PeptideEclipse will generate a tab-separated file (open using Excel) with the following columns:</p>

<pre><code>group_number    protein-group number
group_probability   protein-group probability
protein_name    name of first protein
protein_peptides    number of peptides
protein_pct_spectrum_ids    percent of total spectral counts
protein_percent_coverage    percent of protein sequence coverage
peptide_sequence    stripped peptide sequence
peptide_sequence_modified   modified peptide sequence
peptide_charge  peptide charge state
peptide_nsp_adjusted_probability    peptide probability
peptide_n_enzymatic_termini number of proteolytic termini
peptide_calc_neutral_pep_mass   neutral theoretical peptide mass
proteinID   matching UniProt ID
protein_transmemRegions UniProt transmembrane domain count
protein_AA  length of protein
protein_AA_covered  length of protein covered by peptides
protein_AA_transmem length of protein's TM region(s)
protein_AA_transmem_covered length of protein's TM region(s) covered by peptides
peptide_AA  length of peptide
peptide_AA_transmem length of peptide's TM region(s)
</code></pre>

<p>For a <code>prot.xls</code> file input, PeptideEclipse will create a copy of the input file with the following columns appended:</p>

<pre><code>peptide_AA  length of peptide
peptide_AA_transmem length of peptide's TM region(s)
proteinID   matching UniProt ID
protein_transmemRegions UniProt transmembrane domain count
protein_AA  length of protein
protein_AA_covered  length of protein covered by peptides
protein_AA_transmem length of protein's TM region(s)
protein_AA_transmem_covered length of protein's TM region(s) covered by peptides
</code></pre>

<h3>
<a name="dependencies" class="anchor" href="#dependencies"><span class="octicon octicon-link"></span></a>Dependencies</h3>

<p>All dependencies are packed within the <code>PeptideEclipse.jar</code> file. No external dependencies are needed!
Internally, PeptideEclipse depends on two libraries:</p>

<ul>
<li>Apache Commons CLI: <a href="http://commons.apache.org/proper/commons-cli/">http://commons.apache.org/proper/commons-cli/</a>
</li>
<li>Google Guava: <a href="http://code.google.com/p/guava-libraries/">http://code.google.com/p/guava-libraries/</a>
</li>
</ul><h3>
<a name="credits" class="anchor" href="#credits"><span class="octicon octicon-link"></span></a>Credits</h3>

<p>PeptideEclipse was developed by Ulrich Omasits, and will be published as part of a proteomics study.
Please cite xxx.</p>

<h3>
<a name="license" class="anchor" href="#license"><span class="octicon octicon-link"></span></a>License</h3>

<p>PeptideEclipse is licensed under a Creative Commons Attribution 3.0 Unported License.</p>
      </section>
    </div>

    <!-- FOOTER  -->
    <div id="footer_wrap" class="outer">
      <footer class="inner">
        <p class="copyright">PeptideEclipse maintained by <a href="https://github.com/ulo">ulo</a></p>
        <p>Published with <a href="http://pages.github.com">GitHub Pages</a></p>
      </footer>
    </div>

              <script type="text/javascript">
            var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
            document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
          </script>
          <script type="text/javascript">
            try {
              var pageTracker = _gat._getTracker("UA-42402906-1");
            pageTracker._trackPageview();
            } catch(err) {}
          </script>


  </body>
</html>
